### ISSUE TITLE
Cannot Change to English keyboard

### ISSUE THREAD

**Issue by nixzoom** [smc/Indic-Keyboard#41]
After updating to indic keyboard verson 2.0 i have found an issue. When I change to english keyboard by pressing globe sign in left to the space bar, the keyboard fall back to LG built in keyboard. I have to re-enable it for using it again by going to indic keyboard app. Is there any work around ?
OS Lollipop 5.0
LG G3.

**Comment by jishnu7** [comment]
sorry. you have to select languages/layouts again from settings. otherwise, globe button won't switch layouts. This is one time thing.

**Comment by jishnu7** [comment]
also, there is an option to disable switching to other keyboard in 'Multilingual settings' under Indic Keyboard settings.

**Comment by nixzoom** [comment]
Sorry, it didnt work out. Globe button fails to switch back to English after I finished with Malayalam (Inscrypt). Second, I could not find an option in for Disable switching to other Keyboard under Indic keyboard setting. Even, there is no Multilingual settings'.

**Comment by jishnu7** [comment]
which version of android are you using?

**Comment by nixzoom** [comment]
Android 5.0
Kernel 3.4.0

**Comment by nixzoom** [comment]
For Your Information, I had tried the older version, i mean release just before 2.0. And it works as expected. I think something buggy in 2.0 for Lollipop at least for LG G3.

**Comment by stultus** [comment]
[Reported by KiranZ on [facebook](https://www.facebook.com/kiranzz/posts/10153474148001064?comment_id=10153474182761064&offset=0&total_comments=30&comment_tracking=%7B%22tn%22%3A%22R%22%7D)  ]

 ഇൻഡിക്കിൽ ടൈപ്പ് ചെയ്തോണ്ടിരിക്കുന്ന ഞാൻ.ദാ കീബോർഡ് മാറ്റാനുള്ള ഗ്ലോബ് ബട്ടണിൽ ക്ലിക്കുന്നു
..now I have English keyboard and it also gives me no option to toggle back to Indic. ..I need to go again to the default keyboard settings and to change the default keyboard as Indic..real pain..phone model Samsung Note 4..Lollipop

**Comment by stultus** [comment]
@jishnu7  is it possible to disable switching to other keyboards by clicking on the globe icon?

**Comment by gnuanu** [comment]
@stultus, The usual behaviour of globe button is to switch between enabled
layouts within the selected keyboard. Long pressing the globe will open a
dialog from where you can select other keyboards. From the description
given in the report, it seems like a but more specific to the model.

On Mon, Jul 20, 2015 at 1:21 PM, Hrishi <notifications@github.com> wrote:

> @jishnu7 <https://github.com/jishnu7> is it possible to disable switching
> to other keyboards by clicking on the globe icon?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/smc/Indic-Keyboard/issues/41#issuecomment-122788898>.
>



-- 
അനൂപ് പനവളപ്പില്‍

**Comment by asdofindia** [comment]
I think this is because of a non-recommended practice from Samsung keyboard.
Android [documentation says](https://developer.android.com/guide/topics/text/creating-input-method.html#GeneralDesign)

>  Because multiple IMEs may be installed on the device, provide a way for the user to switch to a different IME directly from the input method UI. 

Any bug on our part can be ruled out by the following:
* Uninstall Indic
* Install any other keyboard, say Google keyboard
* Switch methods using globe
* See if it's possible to return from the Samsung keyboard

**Comment by anivar** [comment]
Issue is a bit different 
As per kiranz says 
ഇത് നോക്കൂ അനിവർ..ഇതാണെന്റെ‌കീബോർഡ്..ഇതിലെ ഗ്ലോബിൽ ക്ലിക്കിയാൽ ഇംഗ്ലീഷ് വരണം..
![11755170_10153474250056064_9059238112773103767_n](https://cloud.githubusercontent.com/assets/422943/8772350/2d0d7164-2ee9-11e5-86fc-a48ff81d9d54.jpg)
പക്ഷേ സംഭവിക്കുന്നത്..ദാ ഇതാണ്..I just pressed on the globe button and now it turns to this screen..and I don't have the globe anymore to go back to Indic..
![11737809_10153474250151064_726028619993397304_n](https://cloud.githubusercontent.com/assets/422943/8772351/2d11be36-2ee9-11e5-80a4-f5abfc6d4371.jpg)

In short glob button fails . 
device Samsung Galxy Note 4 / android 5.0

**Comment by gnuanu** [comment]
Exactly, as I mentioned in my previous comment, this is something to do with how Samsung(Or any other OEM) handles the Globe key in a different IME.

**Comment by jishnu7** [comment]
temporary fix is there in latest build
